### PR TITLE
Computed tables

### PR DESCRIPTION
**Pull request by Pierre-Tourneux**
This pull request address the issue #9.

### CODE REVIEW

File: kirby/api/__init__.py [hunk 1]
```diff
@@ -44,3 +44,15 @@ def add_destination(self, destination):
...
+class Table:
+    def __init__(self, table=None):
+        if not table:
+            table = []
+        pass
+
+    def add_listener(self, source, position_to_update, params=None):
+        if not params:
+            params = {}
+        pass
```
Comment (ericgazoni) [inline kirby/api/__init__.py]: why keep a `pass` here ?

File: kirby/api/__init__.py [hunk 2]
```diff
@@ -44,3 +44,15 @@ def add_destination(self, destination):
         self._register(
             {"destination_id": self.get_topic_id(destination.topic_name)}
         )
+
+
+class Table:
+    def __init__(self, table=None):
+        if not table:
+            table = []
+        pass
```
Comment (ericgazoni) [inline kirby/api/__init__.py]: why a `pass` here ?

File: tests/tests_api/test_computed_tables.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+
+def process_generator(key):
+    def process_orders(orders):
+        return orders[key]
+
+    return process_orders
+
+
+def test_it_compute_table(kirby_app):
```
Comment (ericgazoni) [inline tests/tests_api/test_computed_tables.py]: This sounds like an integration test already, I'm not able to figure out what's going to come out of creating a table and adding a listener, and ... 

Probably a good idea to narrow down how each component works individually before running this one test, so I have multiple sources to understand what does what

File: tests/tests_api/test_computed_tables.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+            "bread_volume": 1,
+            "bread_price": 1.5,
+            "pain_chocolat_volume": 2,
+            "pain_chocolat_price": 3,
+        }
+    )
+
+    cashregister.send({"bread_volume": 5, "bread_price": 7.5})
+
+    assert t[1:3][1:3] == [[10, 15], [12, 18]]
```
Comment (ericgazoni) [inline tests/tests_api/test_computed_tables.py]: this is 
o b s c u r e

why all this slicing? what are these magic numbers doing here?

**Review by ericgazoni — CHANGES_REQUESTED** [review]

File: kirby/api/__init__.py (hunk 1 above)
Comment (PierreSavatte) [reply kirby/api/__init__.py]: Deleted

File: kirby/api/__init__.py (hunk 2 above)
Comment (PierreSavatte) [reply kirby/api/__init__.py]: Deleted

File: tests/tests_api/test_computed_tables.py (hunk 3 above)
Comment (PierreSavatte) [reply tests/tests_api/test_computed_tables.py]: The tests are now splitted

File: tests/tests_api/test_computed_tables.py (hunk 4 above)
Comment (PierreSavatte) [reply tests/tests_api/test_computed_tables.py]: Indeed the slicing may be useless. It is now deleted.  
Also put test data into fixtures.

**Pull request closed without merging**